src/var/clearInterval.js

```diff
@@ -0,0 +1,3 @@
+define(function() {
+	return window.clearInterval;
```

Comment:
> It does affect it by creating the variable, in some cases it increases the gzipped size a little. Maybe in others it has the opposite effect, I didn't measure.

Sure, I just meant there's no size difference between putting `var document = window.document` at the top of core.js like we used to or use a var module. The latter is better for AMD loading. For this, if it's smaller to use `window.`, that's fine. But I agree we can do that separately.